### ISSUE TITLE
Allow users to override tuning parameters when calling public methods

### ISSUE THREAD

**Issue by kiudee** [kiudee/cs-ranking#18]
Currently variables like `learning_rate` are specified as a tunable and it is not possible to continue training with a lower learning rate.

Thoughts?

**Comment by prithagupta** [comment]
I mean the user can set the range of the tunable parameters as he wants

**Comment by prithagupta** [comment]
#26 Includes it
